src/assets/TokenListController.ts

```diff
@@ -180,6 +152,9 @@ export class TokenListController extends BaseController<
    * Start polling for the token list.
    */
   async start() {
+    if (!isTokenDetectionEnabledForNetwork(this.chainId)) {
```

Comment:
Should there be a test for this? I'm not sure I saw one.

Reply:
Updated `TokenListController` tests to test for this